pkg/volume/volume.go

```diff
@@ -33,6 +33,9 @@ type Volume interface {
 
 	// MetricsProvider embeds methods for exposing metrics (e.g. used,available space).
 	MetricsProvider
+
+	// GetPropagationMode returns the propagation mode the volume is mounted to.
+	GetPropagationMode() api.PropagationMode
```

Comment:
Why do we need this? Can propagation parameter be read on Attributes? 
Introducing this interface cause update in every volume source, which I believe should be avoided.